queries/nu/highlights.scm

```diff
@@ -0,0 +1,396 @@
+; ---
+; keywords
+[
+  "def"
+  "alias"
+  "export-env"
+  "export"
+  "extern"
+  "module"
+  "let"
+  "let-env"
+  "mut"
+  "const"
+  "hide-env"
+  "source"
+  "source-env"
+  "overlay"
+  "register"
+  "loop"
+  "while"
+  "error"
+  "do"
+  "if"
+  "else"
+  "try"
+  "catch"
+  "match"
+  "break"
+  "continue"
+  "return"
+] @keyword
+
+(hide_mod
+  "hide" @keyword)
+
+(decl_use
+  "use" @keyword)
```

Comment:
Also should  get `@keyword.import`. Also I think it would be nice to have a highlight for the module itself, of `@module`